commit f66041eb0cc5f8ddc590042e96785c18bd30aee8
Author: Till Hofmann <hofmann@kbsg.rwth-aachen.de>
Date:   Wed Dec 8 17:42:09 2021 +0100

    Properly initialize vector of ints timestampVars
    
    This avoids a failing assertion due to the vector being empty when
    assigning to timestampVars[0].

diff --git a/src/popf2/lpscheduler.cpp b/src/popf2/lpscheduler.cpp
index b8ab8b5..341eda1 100644
--- a/src/popf2/lpscheduler.cpp
+++ b/src/popf2/lpscheduler.cpp
@@ -2200,7 +2200,7 @@ LPScheduler::LPScheduler(const MinimalState & theState,
         
     }
 
-    timestampVars.reserve(tsVarCount);
+    timestampVars = std::vector<int>(tsVarCount, 0);
 
     vector<EndDetails> imaginaryMinMax(tsVarCount);
 
diff --git a/src/popf2/lpscheduler.h b/src/popf2/lpscheduler.h
index 9c9b992..f396284 100644
--- a/src/popf2/lpscheduler.h
+++ b/src/popf2/lpscheduler.h
@@ -31,7 +31,7 @@ public:
     static bool optimiseOrdering;
     
 protected:
-    int tsVarCount;
+    unsigned int tsVarCount;
     ChildData * cd;
 
     struct ConstraintPtrLT;
